docs/source/counts.rst

```diff
@@ -1,106 +0,0 @@
-Counting Functions
-=====================
-
-The package provides functions to count the occurences of distinct values.
-
-Counting over an Integer Range
---------------------------------
-
-.. function:: counts(x, a:b[, wv])
-
-    Count the number of times (or total weights if a weight vector ``wv`` is given) values in ``a:b`` appear in array ``x``. Here, the optional argument ``wv`` should be a weight vector of type ``AbstractWeights`` (see :ref:`weightvec`).
-
-    This function returns a vector ``r`` of length ``n``, with ``n = length(a:b) = b-a+1``. In particular, we have
-
-        ``r[i] == countnz(x .== a+(i-1))``
-
-    **Convenient forms:**
```

Comment:
Makes sense, I hadn't noticed it. I've added the multiple forms to the signature for a general `counts` docstring, which is more standard than providing a table.